packages/chart/src/test/sizeHelpers.test.ts

```diff
@@ -0,0 +1,80 @@
+import { calcInitialHeight, getOrientation } from '../helpers/sizeHelpers'
```

Comment:
can you create a new folder in the helpers folder called 'tests' and place this test there? it's easier to find that way.